docs/user-docs/latest/settings/review-workflows.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Review Workflows - Strapi User Guide
+description: Managing your review workflows and stages in Strapi.
+canonicalUrl: https://docs.strapi.io/user-docs/latest/settings/review-workflows.html
+---
+
+# Managing Review Workflows <GoldBadge withLinkIcon link="https://strapi.io/pricing-self-hosted" /> <AlphaBadge />
+
+The Review Workflows feature allows you to create and manage any desired review stages for your content, enabling your team to collaborate in the content creation flow from draft to publication. Configured review workflows are available in the [Content Manager](../content-manager/introduction-to-content-manager.md) and [Content-Types Builder](../content-types-builder/introduction-to-content-types-builder.md).
```

Comment:
What do you think about linking directly to "Reviewing content" for the CM docs?